lib/alf/rest/test/ext.rb

```diff
@@ -1,7 +1,13 @@
 module Rack
   module Test
     class Session
-      attr_reader :headers
+      def headers
+        @env.transform_keys { |k| k.split('_').map { |h| h.capitalize }.join('-') }
+      end
```

Comment:
Is this even used anywhere ? 🤔 I doubt we were reading any other header than content type.